tests/test_metric_common.py

```diff
@@ -152,6 +152,7 @@ def wrapper(patcher):
 # --------------------------------
 
 
+@pytest.mark.integration
```

Comment:
If all the tests in this module are going to be marked as "integration", you could do it only once at the module level:
```python
pytestmark = pytest.mark.integration
```
and remove all integration mark decorators.